en/components/select.md

```diff
@@ -0,0 +1,370 @@
+
+### Event emitting
+Since `igx-select` extends `igx-drop-down`, it also makes good use of its events which include:
+
+#### Opened event
+- Emitted when the drop-down is fully opened.
+
+You can make use of the `onOpened` event like so:
+```html
+<igx-select (onOpened)="handleOpened($event)">
+    <igx-select-item [value]="Apple">Apple</igx-select-item>
+</igx-select>
+```
+
+#### Opening/Closing events
+- Emitted on:
+    - input click
+    - select expand/collapse button click (app scenario)
+- Triggered on:
+    - key interaction
+
+The thing about the `onOpening` and `onClosing` events is that they are fired *before* the animation finishes playing, i.e. before the drop-down is fully **opened** or **closed**.
+```html
+<igx-select (onOpening)="handleOpening($event)" (onClosing)="handleClosing($event)">
+    <igx-select-item [value]="Apple">Apple</igx-select-item>
+</igx-select>
+```
+
+#### Selection event
+- Emitted when the item selection is changing. It is emitted *before* the selection completes.
+-  Emitted when an item is selected by a mouse click.
+-  Emitted when an item is selected by `Enter`, `Space` keys.
+-  Emitted when setting the value property.
+-  Emitted when setting an item's `selected` property.
+
+```html
+<igx-select (onSelection)="handleSelection($event)">
+    <igx-select-item [value]="Apple">Apple</igx-select-item>
+    <igx-select-item [value]="Pear">Pear</igx-select-item>
+</igx-select>
+```
+
+#### Closed event
+- Emitted on clicking outside of the component, when the drop-down is fully closed.
+
+You can make use of the `onClosed` event like so:
+```html
+<igx-select (onClosed)="handleClosed($event)">
+    <igx-select-item [value]="Apple">Apple</igx-select-item>
+</igx-select>
+```
+
+You put all your *handler* functions inside of your *class*:
+```ts
+export class MyClass {
+    /* --- */
+    private handleOpening(event: any): void {
+        // do something
+    }
+    /* --- */
+    private handleSelection(event: any): void {
+        // do something
+    }
+    /* --- */
+    private handleClosed(event: any): any {
+        // do something
+        // return something
+    }
+    /* --- */
+}
+```
+- Please note that the above examples are for demonstration purposes only and are not meant to abide by any code standards.
+
+### Positioning Strategy
+`igx-select` has its own positioning strategy called the `SelectPositioningStrategy`.
+It extends the [*ConnectedPositioningStrategy*](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/classes/connectedpositioningstrategy.html) and allows `igx-select` to position its drop-down list in different ways, relative to the input field.
+
+In order to use the `SelectPositioningStrategy`, you first have to import it alongside the [*OverlaySettings*](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/interfaces/overlaysettings.html) which are going to use it:
```

Comment:
It's not correct to say in order to use, because the strategy is used by default from the select. Here we configure custom settings which extent the SelectPositionStrategy